src/components/collection-progress-group/helpers/parse-collection-progress.ts

```diff
@@ -133,32 +156,52 @@ function getNextTutorial({
 	isCompleted: boolean
 	progressData: ApiCollectionTutorialProgress[]
 	tutorials: TutorialLite[]
-}) {
-	/**
-	 * If we're not authenticated or have no progress,
-	 * we'll use the first tutorial.
-	 *
-	 * If we do have progress data, we return the first incomplete tutorial,
-	 * or if all tutorials are complete, then the first tutorial.
-	 */
-	let targetTutorial
+}): TutorialLite {
 	if (isCompleted || !progressData) {
-		targetTutorial = tutorials[0]
+		/**
+		 * If we're not authenticated or have no progress,
```

Comment:
Leaned on comments here to describe why we do what we do. Based on 📼 [the vid from this comment](https://github.com/hashicorp/dev-portal/pull/1046#pullrequestreview-1120980519) (props for that btw, felt clear to me!), I think this logic should check out! 🤞